Packs/McAfeeNSM/Integrations/McAfeeNSMv2/McAfeeNSMv2.py

```diff
@@ -0,0 +1,1946 @@
+    """ Returns the addresses from the response, for the human-readable in the command get_rule_object.
+        Args:
+            response: Dict - The response from the API.
+        Returns:
+            The list of addresses.
+    """
+    rule_type = response.get('ruleobjType', '')
+    number = 4 if '4' in rule_type else 6
+    if 'HOST' in rule_type:
+        return response.get(f'HostIPv{number}', {}).get(f'hostIPv{number}AddressList', [])
+    elif 'ADDRESS_RANGE' in rule_type:
+        return response.get(f'IPv{number}AddressRange', {}).get(f'IPV{number}RangeList', [Dict])
+    else:  # 'NETWORK'
+        return response.get(f'Network_IPV_{number}', {}).get(f'networkIPV{number}List', [])
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client, username_n_password: str) -> str:
+    """ Test the connection to McAfee NSM.
+    Args:
+        client: Client - A McAfeeNSM client.
+        username_n_password: str - The string that contains username:password to be encoded.
+    Returns:
+        'ok' if the connection was successful, else throws exception.
+    """
+    try:
+        get_session(client, username_n_password)
+        return 'ok'
+    except DemistoException as e:
+        raise Exception(e.message)
+
+
+def list_domain_firewall_policy_command(client: Client, args: Dict) -> CommandResults:
+    """ Gets the list of Firewall Policies defined in a particular domain.
+    Args:
+        client: client - A McAfeeNSM client.
+        args: Dict - The function arguments.
+    Returns:
+        A CommandResult object with the list of Firewall Policies defined in a particular domain.
+    """
+    domain_id = int(args.get('domain_id', None))
+    limit = arg_to_number(args.get('limit', 50)) or 50
+    page = arg_to_number(args.get('page'))
+    page_size = arg_to_number(args.get('page_size'))
+    if (page and not page_size) or (not page and page_size):
+        raise Exception('Please provide both page and page_size arguments.')
+    if args.get('limit') and page:
+        raise Exception('Please provide either limit argument or page and page_size arguments.')
+
+    response = client.list_domain_firewall_policy_request(domain_id)
+    result = sorted(response.get('FirewallPoliciesForDomainResponseList', []), key=lambda k: k['policyId'], reverse=True)
+    result = pagination(result, limit, page, page_size)
+    human_readable = []
+    for value in result:
+        d = {'policyId': value.get('policyId'),
+             'policyName': value.get('policyName'),
+             'domainId': value.get('domainId'),
+             'visibleToChild': value.get('visibleToChild'),
+             'description': value.get('description'),
+             'isEditable': value.get('isEditable'),
+             'policyType': value.get('policyType'),
+             'policyVersion': value.get('policyVersion'),
+             'lastModUser': value.get('lastModUser')}
+        human_readable.append(d)
+
+    hr_title = 'Firewall Policies List'
```

Comment:
You can write the hr_title directly in the readable_output (same for all commands if it short)